powerline/segments/common/players.py

```diff
@@ -528,3 +528,61 @@ def get_player_status(self, pl):
 
 {0}
 ''').format(_common_args.format('itunes')))
+
+
+class MocPlayerSegment(PlayerSegment):
+	def get_player_status(self, pl):
+		'''Return Music On Console (mocp) player information.
+
+		mocp -i returns current information i.e.
```

Comment:
1. Missing double backticks around `mocp -i`.
2. Below is not how code is formatted in rst, you need

```ReST
…

.. code-block::

   <code here, I align it with first `c` from `code-block`>

…
```